packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -712,6 +719,19 @@ sortAndMergeCrsEntries (const Teuchos::ArrayView<size_t> &CRS_rowptr,
   return sortAndMergeCrsEntries(CRS_rowptr, CRS_colind, CRS_vals);
 }
 
+template<class rowptr_view_type, class colind_view_type, class vals_view_type>
+void
+sortAndMergeCrsEntries(rowptr_view_type& CRS_rowptr,
+		       colind_view_type& CRS_colind,
+		       vals_view_type& CRS_vals)
+{
+  using execution_space = typename vals_view_type::execution_space;
+
+  KokkosSparse::sort_and_merge_matrix<execution_space, rowptr_view_type,
+				      colind_view_type, vals_view_type>(CRS_rowptr, CRS_colind, CRS_vals,
+									CRS_rowptr, CRS_colind, CRS_vals);
```

Comment:
This looks like a cut-and-paste error.

Reply:
@jhux2 This looks OK to me, the output views will be shallow-copied to replace the inputs here.

Also @lucbv the new overloads can infer all the template params.

Reply:
@brian-kelley I see this now from the signature.  The call looked strange to me, thanks!